dist/js/shepherd.js

```diff
@@ -157,12 +173,33 @@ var Step = (function (_Evented) {
         }
       }
 
-      if (!this.options.buttons) {
+      // Button configuration
+
+      var buttonsJson = JSON.stringify(this.options.buttons);
```

Comment:
Note to self - this conditional stuff might be able to be abstracted to other places, but this is the only area that uses "python"-y truthy/falsey rules right now.